MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -0,0 +1,567 @@
+    <imageEditor:SfImageEditor.Toolbars>
+        <imageEditor:ImageEditorToolbar Orientaion="Vertical"
+                                        Position="End">
+            <imageEditor:ImageEditorToolbar.ToolbarItems>
+                <imageEditor:ImageEditorToolbarItem Name="Text">
+                    <imageEditor:ImageEditorToolbarItem.SubToolbars>
+                        <imageEditor:ImageEditorToolbar>
+                            <imageEditor:ImageEditorToolbar.ToolbarItems>
+                                <imageEditor:ImageEditorToolbarItem Name="AddText" />
+                                <imageEditor:ImageEditorToolbarItem Name="TextColor" />
+                                <imageEditor:ImageEditorToolbarItem Name="Delete" />
+                            </imageEditor:ImageEditorToolbar.ToolbarItems>
+                        </imageEditor:ImageEditorToolbar>
+                    </imageEditor:ImageEditorToolbarItem.SubToolbars>
+                </imageEditor:ImageEditorToolbarItem>
+                <imageEditor:ImageEditorToolbarItem Name="Shape" />
+                <imageEditor:ImageEditorToolbarItem Name="Pen" />
+            </imageEditor:ImageEditorToolbar.ToolbarItems>
+        </imageEditor:ImageEditorToolbar>
+    </imageEditor:SfImageEditor.Toolbars>
+</imageEditor:SfImageEditor>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+imageEditor.AutoGenerateToolbarItems = false;
+ImageEditorToolbar editorToolbar = new ();
+editorToolbar.Orientaion = ToolbarOrientation.Vertical;
+editorToolbar.Position = ToolbarPosition.End;
+editorToolbar.ToolbarItems = new List<IImageEditorToolbarItem>()
+{
+    new ImageEditorToolbarItem(){ Name = "Text",
+        SubToolbars = new List<ImageEditorToolbar>()
+        {
+            new ImageEditorToolbar()
+            {
+                ToolbarItems = new List<IImageEditorToolbarItem>()
+                {
+                   new ImageEditorToolbarItem(){ Name = "AddText"},
+                   new ImageEditorToolbarItem(){ Name = "TextColor"},
+                   new ImageEditorToolbarItem(){ Name = "Delete"}
+                },
+            }
+        }
+    },
+    new ImageEditorToolbarItem(){ Name = "Shape"},
+    new ImageEditorToolbarItem(){ Name = "Pen"}
+};
+
+imageEditor.Toolbars.Add(editorToolbar);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## ToolbarItemSelected event 
+
+Whenever you tap the toolbar menu item, the `ToolbarItemSelected` event will be triggered, and you can get the respective tapped toolbar item as an argument as shown in the following code snippet. 
+
+Default toolbar item action can be restricted by setting `e.cancel` to `true`.
+
+{% tabs %}
+
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+    <imageEditor:SfImageEditor Source="image.png" ToolbarItemSelected = "OnImageEditorToolbarItemSelected" />
+
+{% endhighlight %}
```

Comment:
Remove Upcoming feature mentioned
Built in support to browse images from local device storage.
Built in support to save images using folder picker in device storage.
Localization support for the image editor toolbar, allowing you to use it in your preferred language.
Support to toolbar customization.

Reply:
Removed